components/footer.py

```diff
@@ -45,64 +49,77 @@ def my_footer():
+                                        maw=45,
+                                        alt="linkedin logo",
+                                    ),
+                                    href="#",
                                 ),
-                                href="#",
-                            )
+                            ],
+                            align="center",
+                            justify="center",
+                            gap="xl",
                         ),
-                        span={"base": 6, "md": 1},
+                        span={"base": 12, "md": 2},
                     ),
                     dmc.GridCol(
-                        children=[
-                            dmc.Text(
-                                TextFooter.cite_strong.value,
-                                fw=700,
-                                c="white",
-                                size="sm",
-                            ),
-                            dmc.Text(
-                                TextFooter.cite.value,
-                                c="white",
-                                size="sm",
-                            ),
-                            dmc.Anchor(
+                        dmc.Stack(
+                            [
                                 dmc.Text(
-                                    TextFooter.cite_link.value,
+                                    TextFooter.cite_strong.value,
+                                    fw=700,
                                     c="white",
                                     size="sm",
-                                    td="underline",
                                 ),
-                                href=TextFooter.cite_link.value,
-                            ),
-                        ],
-                        p="1rem",
+                                dmc.Text(
+                                    [
+                                        dmc.Text(
+                                            TextFooter.cite.value,
+                                            c="white",
+                                            size="sm",
+                                            display="inline",
+                                        ),
+                                        dmc.Anchor(
+                                            TextFooter.cite_link.value,
+                                            c="white",
+                                            size="sm",
+                                            td="underline",
+                                            display="inline",
+                                            href=TextFooter.cite_link.value,
+                                        ),
+                                    ],
+                                ),
+                            ],
+                            align="center",
+                            justify="center",
+                            mt=5,
```

Comment:
instead of using mt and mb you can simply use `my=5` I also do not like that we have fixed margins in pixel, I think it is better to use marging in `rem` since they will be adjusted as a function of the text size

Reply:
I fix like this 
![image](https://github.com/user-attachments/assets/1a263629-a075-4132-aa81-393fa5d4ba38)